data/brands/amenity/bank.json

```diff
@@ -7561,18 +7561,30 @@
       }
     },
     {
-      "displayName": "State Employees Credit Union",
+      "displayName": "State Employees Credit Union (North Carolina)",
       "id": "stateemployeescreditunion-ea2e2d",
       "locationSet": {"include": ["us"]},
       "tags": {
         "amenity": "bank",
         "brand": "State Employees Credit Union",
         "brand:wikidata": "Q7603196",
-        "brand:wikipedia": "en:State Employees Credit Union",
+        "brand:wikipedia": "en:State Employees Credit Union (North Carolina)",
```

Comment:
This is going to get corrected back to the original since there's a redirect to just `en:State Employees Credit Union (North Carolina)",`